packages/ui/src/components/AppContent/index.module.scss

```diff
@@ -69,7 +69,7 @@ body {
     min-height: 640px;
     position: relative;
     padding: _.unit(6);
-    border-radius: var(--radius);
-    box-shadow: var(--shadow-3);
+    border-radius: 16px;
```

Comment:
i think actually it can be bigger, like 24

Reply:
not a blocker though

Reply:
checked with GM aligned cross all the Design System , keep 16 for now.